net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala xdg
+
+DESCRIPTION="GNOME framework for accessing online accounts"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
+
+LICENSE="LGPL-2+"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="debug gnome gtk-doc +introspection kerberos man ms-graph +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+# libsoup used in goaoauthprovider
+# goa kerberos provider is incompatible with app-crypt/heimdal, see
+# https://bugzilla.gnome.org/show_bug.cgi?id=692250
+# json-glib-0.16 needed for bug #485092
+RDEPEND="
+	>=dev-libs/glib-2.67.4:2
+	sys-apps/dbus
+	introspection? ( >=dev-libs/gobject-introspection-0.6.2:= )
+	>=gui-libs/libadwaita-1.4
+	>=gui-libs/gtk-4.10:4
+	>=net-libs/webkit-gtk-2.33.1:4.1
```

Comment:
webkit requirement was dropped at 3.49.0 version

Reply:
Done in https://github.com/gentoo/gentoo/commit/20171033b2493ec07f0e848030c3200b6e3d66fb